src/docs/release-notes/R17/R17.mdx

```diff
@@ -0,0 +1,24 @@
+### Stickersheet Components
+
+-   Added PX Blue components (EmptyState, ChannelValue, Heroes, ListItemTag, InfoListItem, NavRail)
+-   Added more Material UI components (Bottom Sheets, Chips, Snackbars, Cards, Sliders)
+
+### Design Patterns
+
+-   Re-organized and added new content to explain the patterns
+-   Added more images and design specifications.
+-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a single example repository.
+-   Added documentation around accessibility standards and how to design websites and illustrations.
```

Comment:
I would either pop this out into a separate section around Documentation Site, or stick it under miscellaneous (these aren't technically design patterns)

Reply:
I think this makes sense; are there public links to these resources we can place here?

Reply:
Jeff is building the first two right now, but you can add the links to them `/design/accessibility` and `/design/websites` and the other one is `style/illustrations`

Reply:
Got it, thank you 👍